psycopg/psycopgmodule.c

```diff
@@ -899,6 +914,7 @@ INIT_MODULE(_psycopg)(void)
     /* set some module's parameters */
     PyModule_AddStringConstant(module, "__version__", PSYCOPG_VERSION);
     PyModule_AddStringConstant(module, "__doc__", "psycopg PostgreSQL driver");
+    PyModule_AddIntConstant(module, "__libpq_version__", PG_VERSION_NUM);
```

Comment:
I think we may just use strtol(PG_VERSION_HEX, NULL, 0) here and avoid introducing PG_VERSION_NUM?